FHEM/10_SD_GT.pm

```diff
@@ -15,44 +15,44 @@ use warnings;
 use GPUtils qw(GP_Import GP_Export);
 use FHEM::Meta;
 
-our $VERSION = '1.1';
+our $VERSION_SD_GT = '1.2';
```

Comment:
Die Version 2 mal anzugeben halte ich nicht für sinnvoll. Ich würde das hier weglassen. Einmal in den Metadaten sollte reichen.

Reply:
Hat denn schon jemand mal getestet, wenn man nicht die Voraussetzungen (Pakete) für die Meta Daten hat oder ist das in FHEM implementiert?

Reply:
Nein, aber da die Werte automatisch ohne irgendwelche Zusatzinstallationen erschienen, gehe ich davon aus.

Reply:
Dann werde ich mal die Dopplung entfernen.
Als Gedanke habe ich ja das Schema "v24.1.3" weil wenn man nur "v1.2.3" nehmen würde, so könnte man darüber debattieren wann erhöht man dies ... Somit könnte man einfach das Datum nutzen.

Reply:
Genau, das Datum wird ja eh schon aus der ersten Zeile des Modules übernommen:

`FVERSION   14_SD_UT.pm:v1.0.8-s0/2023-12-10`